Sources/SnapshotTesting/Strategy/Any.swift

```diff
@@ -1,8 +1,8 @@
 import Foundation
 
-extension Strategy {
-  public static var any: Strategy<A, String> {
-    return Strategy.lines.pullback { snap($0) }
+extension Strategy where B == String {
```

Comment:
I've gone through and constrained all the extensions. The pullbacks become a lil more verbose, but the benefit in reusing names seems worth it.